src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,20 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="documentTableRow__actions">
+    <a
+      class="kuiLink documentTableRow__action"
+      data-test-subj="docTableRowAction"
+      ng-href="{{ getContextAppHref() }}"
+      ng-if="indexPattern.hasTimeField()"
+    >
+      View surrounding entries
```

Comment:
Can we also change all references to "entries" to instead refer to "documents"? I think this is more consistent with the language Kibana uses elsewhere.